src/utils/temp.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import path from 'path';
+
+function platformTempFolder(): string {
```

Comment:
given that this file only handles temp folders, I would expect it to be named `temp_folder.ts` or similar. `temp.ts` is too generic